src/System.Runtime.Serialization.Xml/tests/XmlDictionaryWriterTest.cs

```diff
@@ -107,7 +107,8 @@ public static void XmlBaseWriter_CheckAsync_ThrowInvalidOperationException()
             var t2 = Assert.ThrowsAsync<InvalidOperationException>(() => writer.WriteBase64Async(bytes, 0, byteSize));
 
             InvalidOperationException e = t2.Result;
-            Assert.StrictEqual(e.Message, "An asynchronous operation is already in progress.");
+            bool isAsyncIsRunningException = e.Message.Contains("XmlAsyncIsRunningException") || e.Message.Contains("in progress");
```

Comment:
What if the exception message is localized?  Should we even be testing the content of the message against a particular string?